qgis-app/templates/base.html

```diff
@@ -64,7 +64,7 @@
                             {% if user.is_authenticated %}
                             <li><a href="{% url "logout" %}">{% trans "Logout" %}</a></li>
                             {% else %}
-                            <li><a href="{% url "login" %}">{% trans "Login" %}</a></li>
+                            <li><a href="{% url "login" %}">{% trans "Login to Publish" %}</a></li>
```

Comment:
How about just "Publish" ?

Reply:
Assuming they will read it. Would a "Search" button *before* the "Publish" one help ?

Reply:
> Assuming they will read it. Would a "Search" button _before_ the "Publish" one help ?

i was thinking after the input field and before the login button

![Screen Shot 2021-04-13 at 09 14 06](https://user-images.githubusercontent.com/1137152/114324791-72862080-9b6f-11eb-833d-67e8288fa56a.png)

Reply:
> How about just "Publish" ?

I thought there may be other reasons to login so i didn't want to remove the word "login"

Reply:
Yes, for reviewing, editing existing plugin, mark as deprecated a plugin etc.
So I think we should keep "login".

But maybe change this PR to only add a "search button".

@jayenashar Good catch for the UX issue !

Reply:
will close this PR and make a new one with a search button (when i have a chance)